docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,444 @@
+
+```scala mdoc:silent
+class Example3 extends RawModule {
+  val incoming = IO(Flipped(new MixedAlignmentBundle))
+  val outgoing = IO(new MixedAlignmentBundle)
+  outgoing.alignedChild := DontCare // Otherwise FIRRTL throws an uninitialization error
+  outgoing :>= incoming
+}
+```
+
+This generates the following Verilog, where the aligned members are ignore and the flipped members are driven `outgoing` to `incoming`:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example3())
+```
+
+> Note: Astute observers will realize that semantically `c :<>= p` is exactly equivalent to `c :<= p` followed by `c :>= p`.
+
+### Coercing mono-direction connection operator (:#=)
+
+For connections where you want to every producer member to always drive every consumer member, regardless of alignment, use `:#=` (the "coercion connection"). This operator is useful for initializing wires whose types contain members of mixed alignment.
+
+```scala mdoc:silent
+import chisel3.experimental.BundleLiterals._
+class Example4 extends RawModule {
+  val w = Wire(new MixedAlignmentBundle)
+  dontTouch(w) // So we see it in the output verilog
+  w :#= (new MixedAlignmentBundle).Lit(_.alignedChild -> true.B, _.flippedChild -> true.B)
+}
+```
+
+This generates the following Verilog, where all members are driven from the literal to `w`, regardless of alignment:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example4())
+```
+
+> Note: Astute observers will realize that semantically `c :#= p` is exactly equivalent to `c :<= p` followed by `p :>= c` (note `p` and `c` switched places in the second connection).
+
+## Waived Data
+
+It is not uncommon for a user to want to connect Chisel components which are not type equivalent. For example, a user may want to cook up the `ready`/`valid` members of a `ReadyValidIO` to a `DecoupledIO`, but
+because the `bits` member is not present in both, our operators would reject a connection.
+
+`WaivedData` is the mechanism to specialize connection operator behavior in these scenarios. For any addition member which is not present in the other component being connected to, they can be explicitly waived from the operator to be ignored, rather than trigger an error.
+
+### Connecting sub-types to super-types by waiving extra members
+
+In the following example, we can use `:<>=` to connect a `MyReadyValid` to a `MyDecoupled` by waiving the `bits` member.
+
+```scala mdoc:silent
+class MyReadyValid extends Bundle {
+  val valid = Bool()
+  val ready = Flipped(Bool())
+}
+class MyDecoupled extends MyReadyValid {
+  val bits = UInt(32.W)
+}
+class Example5 extends RawModule {
+  val in  = IO(Flipped(new MyDecoupled()))
+  val out = IO(new MyReadyValid())
+  out :<>= in.waive(_.bits)
```

Comment:
why does this even work? Do the connectable operators not require exact type matches? I think they should for exactly the reason outlined above...